ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -85,14 +75,14 @@ def _format_single_funnel(self, result, with_breakdown=False):
 
             if result and len(result) > 0:
                 total_people += result[step.order]
-                relevant_people += result[cast(int, step.order) + num_entities]
 
-            serialized_result = self._serialize_step(step, total_people, relevant_people[0:100])
+            serialized_result = self._serialize_step(step, total_people, [])
```

Comment:
Is there any way we can signal why we're not passing in anything for people here? Maybe just a comment?